rasa/core/broker.py

```diff
@@ -17,6 +17,8 @@ def from_endpoint_config(
         return None
     elif broker_config.type == "pika" or broker_config.type is None:
         return PikaProducer.from_endpoint_config(broker_config)
+    elif broker_config.type.lower() == "sql":
```

Comment:
more of a style question, but should we rewrite the `if ... else` stuff to use a dictionary?

E.g.


```python
broker_mapping = {
    "pika": PikaProducer,
    "sql": SQLProducer,
}

broker = broker_mapping.get(xy)
if broker:
   return broker.from_endpoint_config()
else:
  load from module
```

Reply:
In my opinion, we could, but the way it's done right now is not wrong either. Although because each "case" in the if/elif chain is only returning a value and not doing anything else, a `dict` seems a bit more appropiate.

Reply:
👍 Both are fine to me. It was just a suggestion :-)